src/test/java/org/cactoos/http/HtHeadTest.java

```diff
@@ -60,7 +68,7 @@ public void takesHeadOutOfHttpResponse() throws IOException {
                     )
                 )
             ).asString(),
```

Comment:
t@ilyakharlamov please replace the `assertThat` with `Assertion`

Reply:
@victornoel fixed in 8915ec4